disnake/message.py

```diff
@@ -620,21 +655,53 @@ def from_message(cls, message: Message, *, fail_if_not_exists: bool = True) -> S
         ----------
         message: :class:`~disnake.Message`
             The message to be converted into a reference.
+        type: :class:`MessageReferenceType`
+            The type of the message reference. This is used to control whether to reply
+            or forward a message.
+
+            .. versionadded:: 2.10
+
+        channel_id: Optional[:class:`int`]
+            The channel id where the message forward should be sent. This parameter is required to be passed only in case of message forwarding.
+
+            .. versionadded:: 2.10
+
+        guild_id: Optional[:class:`int`]
+            The guild id where the message forward should be sent. This parameter is used in message forwarding. Unlike ``channel_id`` this can be None even when forwarding.
+
+            .. versionadded:: 2.10
```

Comment:
Yeah, at the time of writing this I didn't understand how it worked, then I must have changed it in the code (otherwise my tests wouldn't have worked) but forgot to update the docs. Ty for the catch